api/openapi/my_api.yaml

```diff
@@ -1,66 +1,154 @@
 openapi: 3.0.0
 # Added by API Auto Mocking Plugin
+servers: []
 info:
-  description: Guidance plugin
-  version: "4"
-  title: Guidance plugin
+  description: "The PDS is the main interface to the PDS application for providing clinical decision support for end users. The PDS is a highly fault-tolerant system, meaning that it will attempt to correct errors in order to keep running and avoid catastrophic failure. This means that these endpoints will attempt to return some kind of meaningful data along with 4xx (Client Error) and 5xx (Server Error) status codes. The onus is on the client to check and determine how to render the data returned along with these codes." 
+  version: "5"
+  title: "PDS plugin: primary platform interface"
   contact:
     email: txscience@lists.renci.org
-  license:
-    name: Apache 2.0
-    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 paths:
+  /patientVariables:
```

Comment:
This looks like an endpoint for pds main primary interface, rather than the pdspi-guidance-example endpoint.